### ISSUE TITLE
Ability to decode with command line arguments

### ISSUE THREAD

**Issue by neubig** [neulab/xnmt#290]
It would be nice if we didn't need to make a new YAML file every time we want to decode. I think we should be able to maintain the current behavior, but also have an option to decode with just command line arguments.

**Comment by pmichel31415** [comment]
I'm interested in helping for this, how would I go about doing it?

**Comment by neubig** [comment]
Probably the following would work:
1. Read in the model yaml file to create the experimental configuration.
2. Set "eval_only" in the global parameters.
3. Create a new "DecodingEvalTask" (I think that's the name), and overwrite the inputs to the task with arguments passed on the command line.
4. Run the "experiment".

**Comment by msperber** [comment]
You can also check this example that does something similar: https://github.com/neulab/xnmt/blob/master/examples/10_programmatic_load.py

One small correction is that ```eval_only``` no longer exists, instead you would just set ```train=None```.

**Comment by pmichel31415** [comment]
Can this functionality be documented? Maybe here http://xnmt.readthedocs.io/en/latest/getting_started.html#command-line-tools at least, ideally with examples.

**Comment by msperber** [comment]
Yeah that sounds good, I'll probably get to it next week unless you want to give it a shot before that.

**Comment by pmichel31415** [comment]
It's not urgent and I don't have time to do it right now. Should be done eventually though.
